sunpy/net/dataretriever/sources/kanzelhohe.py

```diff
@@ -0,0 +1,128 @@
+
+
+class KanzelhoheClient(GenericClient):
+    """
+    Returns a list of URLS to Kanzelhohe H-alpha files corresponding to value of input timerange.
+    URL source: `http://cesar.kso.ac.at/`.
+    The earliest data for H-alpha 2k - 20-Jul-2000
+                          Ca-II k - 31-Jul-2010
+                          Continuum - 7-Jan-2011
+
+    Parameters
+    ----------
+    timerange: `~sunpy.time.TimeRange`
+        time range for which data is to be downloaded.
+        Example value - TimeRange('2015-12-30 00:00:00','2015-12-31 00:01:00')
+    Instrument: Fixed argument = 'kanzelhohe'
+    Wavelength: Fixed argument = astropy.units.quantity.Quantity
+                The physical value of wavelength will belong to any of [5460, 6563, 32768]
+                and units will be Angstroms.
+
+    Returns
+    -------
+    urls: `list`
+    list of urls corresponding to requested time range.
+    Examples
+    --------
+    >>> from sunpy.net import Fido, attrs as a
+    >>> timerange = a.Time('2015/12/28 00:00:00','2015/12/28 09:03:00')
+    >>> results = Fido.search(timerange, a.Instrument('kanzelhohe'),
+    ...                       a.Wavelength(6563*u.AA))   #doctest: +REMOTE_DATA
+    >>> print(results)  #doctest: +REMOTE_DATA
+    Results from 1 Provider:
+    <BLANKLINE>
+    1 Results from the KanzelhoheClient:
+         Start Time           End Time      Source   Instrument      Wavelength
+    ------------------- ------------------- ------ -------------- ---------------
+    2015-12-28 00:00:00 2015-12-28 09:03:00    KSO Kanzelhohe HA2 6563.0 Angstrom
+    <BLANKLINE>
+    <BLANKLINE>
+    >>> response = Fido.fetch(results)  #doctest: +SKIP
+    """
+
+    def _get_url_for_timerange(self, timerange, **kwargs):
+        """
+        Return list of URLS corresponding to value of input timerange.
+
+        Parameters
+        ----------
+        timerange: `~sunpy.time.TimeRange`
+            time range for which data is to be downloaded.
+
+        Returns
+        -------
+        urls : `list`
+            list of URLs corresponding to the requested time range
+        """
+        wave = kwargs['wavelength']
+        wave_float = (wave.min if isinstance(wave, a.Wavelength) else wave.min()).value
+        table = {6563: {'datatype': ['halpha2k/recent', 'halph_fr'],
+                        'start_time': datetime.datetime(2000, 7, 20, 7, 45, 46)},
+                 32768: {'datatype': ['caiia', 'caiik_fi'],
+                         'start_time': datetime.datetime(2000, 7, 20, 7, 45, 46)},
+                 5460: {'datatype': ['phokada', 'bband_fi'],
+                        'start_time': datetime.datetime(2000, 7, 20, 7, 45, 46)}}
+        # Checking if value is close enough to a wavelength value.
+        # Converting from one unit to other introduces precision errors.
+        wavelengths = table.keys()
+        wave_list = list(filter(lambda x: np.isclose(x, wave_float), wavelengths))
+        if(len(wave_list) == 0):
```

Comment:
```suggestion
        if len(wave_list) == 0:
```
No need to put the parentheses.